Header header  

uint8 NOTHING=0
uint8 WALKING_AWAY=1
uint8 WALKING_TOWARDS=2
uint8 WALKING_PAST=3

uint8 activity
float32 confidence